models/QtiRunnerInitDataBuilder.php

```diff
@@ -236,6 +238,8 @@ protected function getTestMap(QtiRunnerServiceContext $context, array $itemsStat
                                     self::OUTCOME_VAR_MAXSCORE
                                 ),
                         'isExternallyScored' => $this->isExternallyScored($itemData['data'] ?? []),
+                        //invert the condition to have a better attribute exposed on the FE
+                        'pendingExternalScore' => !($itemsStates[$itemId]['isExternallyGraded'] ?? false),
```

Comment:
Is this ok? If $itemsStates[$itemId]['isExternallyGraded'] is null so no external graded flag, then pendingExternalScore is true? If there's no external flag how we can know that external score is pending or is simply not existing? Little bit confusing

Reply:
yes, because pendingExternalScore should reflect the opposite of isExternallyGraded. isExternallyGraded will be false by default, we can only have null case the db field is not preset or in case of undefined variable, in this case we should consider is as false. isExternallyGraded will be true only if the score is graded externally.

Reply:
Ok if it just fallback to missing field in db then its fine :)